vmdk_plugin/sanity_test.go

```diff
@@ -240,7 +242,7 @@ func TestSanity(t *testing.T) {
 			"size": "1gb",
 		},
 	}
-	// Create/delete routine
+	// Create/delete goroutine
```

Comment:
So the problem here - which I observed on my testbed-  is that if there is  a client  in clients more than once, the code below (go func) fails because ESX rejects reconfigure request on the same VM when another one is in flight. 

and our current Makefiles allow for VM1 and  VM2 (test clients) to be the same - in fact this is how single VM testing is done. 

I'd suggest  checking for this  and warn then skip duplicate clients. Yes, it will skip the parallel test but  will unblock single VM test.